src/Umbraco.Web.UI.Client/src/less/sections.less

```diff
@@ -111,32 +111,43 @@ ul.sections li.help {
 	bottom: 0;
 	left: 0;
 	display: block;
-	width: 100%;
+	width: ~"calc(100% - 5px)"; //subtract 4px orange border + 1px border-right for sections

```

Comment:
Somehow this doesn't compile correct to `width: calc(100% - 5px)` it returns `width: calc(95%)`.